sonar-project.properties

```
@@ -0,0 +1,36 @@
+# Organization and project keys are displayed in the right sidebar of the project homepage
+sonar.organization=owncloud-1
+sonar.projectKey=owncloud_core
+sonar.projectVersion=1.0
+sonar.host.url=https://sonarcloud.io
+
+# =====================================================
+#   Meta-data for the project
+# =====================================================
+
+sonar.links.homepage=https://github.com/owncloud/core
+sonar.links.ci=https://drone.owncloud.com/owncloud/core/
+sonar.links.scm=https://github.com/owncloud/core
+sonar.links.issue=https://github.com/owncloud/core/issues
+
+# =====================================================
+#   Properties that will be shared amongst all modules
+# =====================================================
+
+# Just look in these directories for code
+sonar.sources=.
+sonar.inclusions=apps/**,core/**,lib/**,ocs/**,settings/**
```

```suggestion
sonar.inclusions=appinfo/**,lib/**
```
These look like the dirs that have code that could be analysed.

---

done